operators/virtualhost-operator/VirtualHostOperator.js

```diff
@@ -69,7 +70,8 @@ class VirtualHostOperator extends BaseOperator {
         resourceId: changeObjectBody.metadata.name,
         status: {
           response: response,
-          state: CONST.APISERVER.RESOURCE_STATE.SUCCEEDED
+          state: CONST.APISERVER.RESOURCE_STATE.SUCCEEDED,
```

Comment:
You have not changed the vhost CRD. This would fail then.

Better check it with @pranjaljain